src/android/CHIPTool/app/src/main/java/com/google/chip/chiptool/clusterclient/OnOffClientFragment.kt

```diff
@@ -20,6 +22,7 @@ class OnOffClientFragment : Fragment() {
     get() = ChipClient.getDeviceController()
 
   private var commandType: ChipCommandType? = null
+  private var levelValue: Int? = null
```

Comment:
You seem to use 0 below to indicate "not applicable" state, so consider using non-nullable type "Int!"

Reply:
I should probably use null when On/Off was touched, but I am not 100% sure how to pass that through JNI.